src/com/github/mtdrewski/GRAPH_moment/controller/DirectedEdgeLine.java

```diff
@@ -0,0 +1,62 @@
+package com.github.mtdrewski.GRAPH_moment.controller;
+
+import javafx.beans.binding.DoubleBinding;
+import javafx.scene.shape.Polygon;
+import javafx.scene.transform.Rotate;
+
+
+public class DirectedEdgeLine extends EdgeLine{
+
+    private Polygon triangle;
+    private DoubleBinding dx;
+    private DoubleBinding dy;
+
+    public DirectedEdgeLine(GraphDrawerController drawer) {
+        super(drawer);
+        dx = endXProperty().add(startXProperty().negate());
+        dy = endYProperty().add(startYProperty().negate());
+        triangle = new Polygon(getEndX()+5, getEndY(), getEndX() - 24, getEndY() + 12, getEndX() - 24, getEndY() - 12);
+        var rotate = new Rotate(0,0,0,1,Rotate.Z_AXIS);
+
+        triangle.getTransforms().add(rotate);
+        dx.addListener((observable, oldValue, newValue) -> {
+            rotate.setAngle(getAngle(dy.doubleValue(), newValue.doubleValue()));
+        });
+        dy.addListener((observable, oldValue, newValue) -> {
+            rotate.setAngle(getAngle(newValue.doubleValue(), dx.doubleValue()));
+        });
+
+        triangle.layoutXProperty().bindBidirectional(endXProperty());
+        triangle.layoutYProperty().bindBidirectional(endYProperty());
+    }
+
+    @Override
```

Comment:
It doesn't override anything in edgeline or am I mistaken?

Reply:
It does. Line 55 in EdgeLine